package.xml

```diff
@@ -6,8 +6,17 @@
   <description>gtsam</description>
 
   <maintainer email="gtsam@lists.gatech.edu">Frank Dellaert</maintainer>
+
+  <!-- Main License for GTSAM sources -->
   <license>BSD</license>
-  
+  <!-- 3rd party libraries -->
+  <license file="gtsam/3rdparty/CCOLAMD/Doc/License.txt">BSD</license>
+  <license file="gtsam/3rdparty/ceres/jet.h">BSD</license>
+  <license file="gtsam/3rdparty/Eigen/COPYING.README">MPL2</license>
```

Comment:
```suggestion
  <license file="gtsam/3rdparty/Eigen/COPYING.README">MPL-2.0</license>
```

Use the full [SPDX.org](https://spdx.org/licenses/) license identifier for MPL 2.0.